src/Microsoft.Extensions.Caching.Abstractions/DistributedCacheExtensions.cs

```diff
@@ -43,5 +44,35 @@ public static Task SetAsync(this IDistributedCache cache, string key, byte[] val
 
             return cache.SetAsync(key, value, new DistributedCacheEntryOptions());
         }
+
+        public static void SetString(this IDistributedCache cache, string key, string value)
+        {
+            cache.Set(key, Encoding.UTF8.GetBytes(value));
```

Comment:
From the doc it throws, it needs to be handled then.

https://msdn.microsoft.com/en-us/library/ds4kkd55(v=vs.110).aspx